src/main/resources/database/changes/release-22/add-activeenrolment-to-case.sql

```diff
@@ -0,0 +1,2 @@
+alter table casesvc."case"
+add column activeEnrolment boolean;
```

Comment:
This shouldn't be camelCase, all other columns in the table are lower case. Please change to "activeenrolment" for consistency.

Reply:
Also I thought we were going to put a dash in the file name between active and enrolment?

Reply:
done.